File: fahr-to-cel/farTocel.js
(() => {
  let upper = 300;

  console.log(`Fahr \t Cel\n\n`)

  for (let fahr = 0; fahr <= upper; fahr += 20) {
    let cel = 5 * (fahr - 32) / 9;
    console.log(`${fahr} \t ${parseInt(cel)}`);
  }
})();

File: fahr-to-cel/farTocel.c
#include <stdio.h>

int main() {
  int fahr;
  int cel;
  int upper=300;

  printf("Fahr\t Cel\n\n");

  for(fahr=0; fahr<=upper; fahr+=20) {
    cel = 5 * (fahr - 32) / 9;
    printf("%d\t %d\n", fahr, cel);
  }

  return 0;
}